code/modules/hydroponics/fermenting_barrel.dm

```diff
@@ -50,6 +50,10 @@
 		to_chat(user, "<span class='notice'>You place [I] into [src] to start the fermentation process.</span>")
 		addtimer(CALLBACK(src, .proc/makeWine, fruit), rand(80, 120) * speed_multiplier)
 		return TRUE
+	var/obj/item/W = I
+	if(W)
+		if(W.is_refillable())
+			return 0 //so we can refill them via their afterattack.
```

Comment:
FALSE

Reply:
do you want me to change reagent dispenser code with that too, since thats where I copied this from?

Reply:
actually doing that would probably fix mickyans issue

Reply:
Not really, only files you touch